### ISSUE TITLE
crash on latest box2d.dart

### ISSUE THREAD

**Issue by kevmoo** [dart-lang/dartdoc#418]
Sync down latest - at https://github.com/google/box2d.dart/commit/6500396c4363f160796b4b74beb1072773fa6ab1

Using dartdoc at latest - https://github.com/dart-lang/dartdoc/commit/705b83b75206dc2ba1a8f62ab930a92041493e3f

```
Generating Dart API docs into docs
parsing /Users/kevmoo/source/github/dbox2d/lib/box2d.dart...
parsing /Users/kevmoo/source/github/dbox2d/lib/box2d_browser.dart...

Parsed 2 libraries in 4.1 seconds.

Unhandled exception:
Uncaught Error: The null object does not have a getter 'iterator'.

NoSuchMethodError: method not found: 'iterator'
Receiver: null
Arguments: []
Stack Trace:
#0      Object._noSuchMethod (dart:core-patch/object_patch.dart:42)
#1      Object.noSuchMethod (dart:core-patch/object_patch.dart:45)
#2      Element.accept (package:markdown/src/ast.dart:39)
#3      PlainTextRenderer.render (package:dartdoc/src/html_generator.dart:592)
#4      oneLiner (package:dartdoc/src/html_generator.dart:546)
#5      _MustacheContext.callLambda (package:mustache4dart/mustache_context.dart:70)
#6      _MustacheContext.call (package:mustache4dart/mustache_context.dart:64)
#7      _StartSectionToken.apply (package:mustache4dart/src/tokens.dart:298)
#8      _StartSectionToken.apply.<anonymous closure>.<anonymous closure> (package:mustache4dart/src/tokens.dart:290)
#9      _StartSectionToken.forEachUntilEndSection (package:mustache4dart/src/tokens.dart:312)
#10     _StartSectionToken.apply.<anonymous closure> (package:mustache4dart/src/tokens.dart:290)
#11     IterableBase.forEach (dart:collection/iterable.dart:229)
#12     _StartSectionToken.apply (package:mustache4dart/src/tokens.dart:289)
#13     Token.call (package:mustache4dart/src/tokens.dart:44)
#14     _Template._write (package:mustache4dart/src/tmpl.dart:100)
#15     _Template.call (package:mustache4dart/src/tmpl.dart:72)
#16     HtmlGenerator._build (package:dartdoc/src/html_generator.dart:490)
#17     HtmlGenerator.generatePackage (package:dartdoc/src/html_generator.dart:207)
#18     HtmlGenerator.generate.<generate_async_body> (package:dartdoc/src/html_generator.dart:126)
#19     _RootZone.runUnary (dart:async/zone.dart:1155)
#20     _Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:494)
#21     _Future._propagateToListeners (dart:async/future_impl.dart:577)
#22     _Future._completeWithValue (dart:async/future_impl.dart:368)
#23     _Future._asyncComplete.<anonymous closure> (dart:async/future_impl.dart:422)
#24     _asyncRunCallbackLoop (dart:async/schedule_microtask.dart:41)
#25     _asyncRunCallback (dart:async/schedule_microtask.dart:48)
#26     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:96)
#27     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:143)

#0      _rootHandleUncaughtError.<anonymous closure> (dart:async/zone.dart:886)
#1      _asyncRunCallbackLoop (dart:async/schedule_microtask.dart:41)
#2      _asyncRunCallback (dart:async/schedule_microtask.dart:48)
#3      _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:96)
#4      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:143)
```

**Comment by kevmoo** [comment]
Also reproduces at `dartdoc 0.0.1+5.`

**Comment by keertip** [comment]
fixed in #428
